ovos_tts_plugin_server/__init__.py

```diff
@@ -1,12 +1,14 @@
 import requests
+import random
 from ovos_plugin_manager.templates.tts import TTS, TTSValidator, RemoteTTSException
 
 
 class OVOSServerTTS(TTS):
-    public_servers = (
+    public_servers = [
         "https://mimic3.ziggyai.online/",
         "https://tts.smartgic.io/mimic3/"
-    )
+    ]
+    random.shuffle(public_servers)  # Spread the load among all public servers
```

Comment:
this only shuffles once, should be done per request instead, otherwise whatever ends up in 1st after the shuffle is still always used

Reply:
This was mostly intentional so we don't have to do as much processing where latency matters. In either case each server should get about the same percentage of requests on average

Reply:
dont think a shuffle will significantly increase latency....